api-catalog-ui/frontend/cypress/integration/e2e/detail-page/detail-page.test.js

```diff
@@ -90,8 +90,6 @@ describe('>>> Detail page test', () => {
 
         cy.contains('Service Homepage').should('exist');
 
-        cy.get('pre.version').should('contain', '1.1.2');
```

Comment:
do you think this check should be removed?

Reply:
Do you think it makes sense to check the API version? I don't see much value in it and you have to increase it when you update the API.

Reply:
Yes, it doesn't have much value to keep it. I wanted just to ask